Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Use the official ROS 2 humble base image
+FROM ros:humble
+
+# Set the working directory
+WORKDIR /root/ros2_ws/src
+
+COPY . .
+# Install additional dependencies
+RUN apt-get update \
+    && apt-get -y --quiet --no-install-recommends install \
+    gcc \
+    git \
+    libxml2-dev \
+    libxslt-dev \
+    python3 \
+    ros-humble-ament-cmake-clang-format\
```

Comment:
This is already installed because I declared a dependency on it in the package.xml.